Sources/Navigator/EPUB/EPUBNavigatorViewController.swift

```diff
@@ -250,15 +252,16 @@ open class EPUBNavigatorViewController: UIViewController,
             httpServer: httpServer
         )
 
-        self.init(viewModel: viewModel, initialLocation: initialLocation)
+        self.init(viewModel: viewModel, initialLocation: initialLocation, readingOrder: readingOrder)
     }
 
     @available(*, deprecated, message: "See the 2.5.0 migration guide to migrate the HTTP server and settings API")
     public convenience init(
         publication: Publication,
         initialLocation: Locator? = nil,
         resourcesServer: ResourcesServer,
-        config: Configuration = .init()
+        config: Configuration = .init(),
+        readingOrder: [Link]? = nil
```

Comment:
This is a deprecated constructor, so you don't need to add the argument here.

```suggestion
        config: Configuration = .init()
```